### ISSUE TITLE
Build macOS PyPy3.5 wheels

### ISSUE THREAD

**Issue by webknjaz** [pyca/cryptography#4659]
We have macOS env with PyPy under Travis CI. Because cryptography doesn't ship matching wheels it requires extra steps/hacks to set up pre-requisites so that `pip install` would succeed.

I think it'd be nice to also publish such wheels as well.

**Comment by alex** [comment]
Unfortunately as of when we last investigated, PyPy does not have a stable enough ABI that we can build a wheel to cover a significant portion of PyPy users. AFAIK we'd need a separate wheel for every single PyPy release. As long as this is the case, we probably won't distribute PyPy wheels.

**Comment by webknjaz** [comment]
True, it might be the case. Isn't it automated though? Also, it's probably the same as with manylinux1, isn't it?
